src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -100,6 +101,52 @@ public void testGetExistingWorkspace() throws Exception {
     assertThat(desc.getId(), equalTo(workspaceId));
   }
 
+  @Test
+  public void testDefaultWorkspaceStage() throws Exception {
```

Comment:
nit: add the expected outcome as well as the scenario if the outcome isn't obvious, e.g. `testDefaultWorkspaceStageIsRawls`